config/config.test.json.sample

```diff
@@ -32,9 +32,9 @@
     }
   },
   "paths": {
-    "collections": "test/acceptance/workspace/collections",
-    "endpoints": "test/acceptance/workspace/endpoints",
-    "hooks": "test/acceptance/workspace/hooks"
+    "collections": "test/acceptance/temp-workspace/collections",
```

Comment:
@eduardoboucas I don't see anything that actually regenerates the temp-workspace? I might have missed it?

Reply:
No, I missed it! 🙈 Added to `package.json` now.